content/documentation/staging/configuration/vm.adoc

```diff
@@ -0,0 +1,43 @@
+---
+title: "Virtual Machine workloads"
+date: 2021-07-26T00:00:00+02:00
+draft: false
+weight: 9
+---
+
+:toc: macro
+:toc-title: In this section:
+:keywords: vm virtual machine
+:icons: font
+:sectlinks:
+
+toc::[]
+
+== Introduction
+
+Kiali supports visualizing Virtual Machine workloads (WorkloadEntry) on the graph the same as pod-based workloads running inside a Kubernetes cluster. You must ensure that Istio Proxy is running and correctly configured on the Virtual Machine and Prometheus must be able to scrape the metrics endpoint of the Istio Proxy running on the VM. From there, Kiali will be able to read the traffic telemetry for the Virtual Machine workloads and incorporate the workloads into the graph.
```

Comment:
suggested rewording:

Kiali graph visualizes both Virtual Machine workloads (WorkloadEntry) and pod-based workloads, running inside a Kubernetes cluster.  You must ensure that the Istio Proxy is running, and correctly configured, on the Virtual Machine.  Also, Prometheus must be able to scrape the metrics endpoint of the Istio Proxy running on the VM.   Kiali will then be able to read the traffic telemetry for the Virtual Machine workloads, and incorporate the VM workloads into the graph.